test/models/viewmodels/notificationspage/converters/ControlResponseConverterSpec.scala

```diff
@@ -153,28 +154,32 @@ class ControlResponseConverterSpec extends BaseSpec with MockitoSugar {
         val input = exampleNotificationFrontendModel(
           responseType = ResponseType.ControlResponse,
           actionCode = Some(ActionCode.Rejected.code),
-          errorCodes = Seq("01", "29", "13")
+          errorCodes = Seq("01", "29", "13", "E408", "E3305")
         )
 
         contentBuilder.convert(input)
 
-        verify(decoderMock).errorCode(meq("01"))
-        verify(decoderMock).errorCode(meq("29"))
-        verify(decoderMock).errorCode(meq("13"))
+        verify(decoderMock).ileErrorCode(meq("01"))
+        verify(decoderMock).ileErrorCode(meq("29"))
+        verify(decoderMock).ileErrorCode(meq("13"))
+        verify(decoderMock).chiefErrorCode(meq("E408"))
+        verify(decoderMock).chiefErrorCode(meq("E3305"))
       }
 
       "return NotificationsPageSingleElement with values returned by Messages" in new Test {
 
         when(decoderMock.actionCode(any[String])).thenReturn(Some(ActionCode.Rejected))
-        when(decoderMock.errorCode(meq("01"))).thenReturn(Some(InvalidUcrFormat))
-        when(decoderMock.errorCode(meq("13"))).thenReturn(Some(NoPriorArrivalFoundAtDepartureLocation))
-        when(decoderMock.errorCode(meq("29"))).thenReturn(Some(MucrAlreadyDeparted))
+        when(decoderMock.ileErrorCode(meq("01"))).thenReturn(Some(InvalidUcrFormat))
+        when(decoderMock.ileErrorCode(meq("13"))).thenReturn(Some(NoPriorArrivalFoundAtDepartureLocation))
+        when(decoderMock.ileErrorCode(meq("29"))).thenReturn(Some(MucrAlreadyDeparted))
+        when(decoderMock.chiefErrorCode("E408"))
+          .thenReturn(Some(CHIEFError("E408", "Unique Consignment reference does not exist")))
```

Comment:
Nit: Could extract the description into a constant and use it in assertion. Then you're sure not having mistype errors.

Reply:
Done